extensions/schema-compare/src/dialogs/schemaCompareOptionsDialog.ts

```diff
@@ -183,15 +183,15 @@ export class SchemaCompareOptionsDialog {
 		this.deploymentOptions = defaultOptions;
 	}
 
-	protected async initializeDialog() {
+	protected async initializeDialog(): Promise<void> {
 		this.generalOptionsTab = azdata.window.createTab(loc.GeneralOptionsLabel);
```

Comment:
does anything inside this function awaitable?

Reply:
oops good catch